blackjax/inference/hmc/metrics.py

```diff
@@ -34,9 +34,7 @@
 EuclideanKineticEnergy = Callable[[PyTree], float]
 
 
-def gaussian_euclidean(
-    inverse_mass_matrix: Array,
-) -> Tuple[Callable, EuclideanKineticEnergy, Callable]:
+def gaussian_euclidean() -> Tuple[Callable, EuclideanKineticEnergy, Callable]:
```

Comment:
I think we should keep the origin signature here that takes `inverse_mass_matrix` as input and dispatch 
```python
dot, matmul = jnp.dot, jnp.matmul
```
in the if-else.